jcloud/normalize.py

```diff
@@ -39,6 +39,35 @@ class FlowYamlNotFound(FileNotFoundError):
     pass
 
 
+def stringify(v: Any) -> str:
+    if isinstance(v, str):
+        return v
+    elif isinstance(v, int) or isinstance(v, float):
+        return str(v)
+
+
+def stringify_labels(flow_dict: Dict) -> Dict:
```

Comment:
Pls add unit tests for this.

Reply:
Just added one let me know if this will suffice.